mobile_app_connector/models/compassion_correspondence.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Nicolas Badoux <n.badoux@hotmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import models, api
+from ..mappings.compassion_correspondence_mapping import \
+    MobileCorrespondenceMapping
+
+
+class CompassionCorrespondence(models.Model):
+    _inherit = 'correspondence'
+
+    @api.model
+    def mobile_post_letter(self, json_data, **parameters):
+        """
+            Mobile app method:
+            POST a letter between a child and a sponsor
+
+            :param parameters: all request parameters
+            :return: sample response
+        """
+        result = "Letter Submitted"
+
```

Comment:
You can remove blank spaces, I'm not huge fan of it especially if the method is so small.